core/ui/src/main/kotlin/com/nexters/bandalart/android/core/ui/component/BottomSheetButton.kt

```diff
@@ -0,0 +1,44 @@
+package com.nexters.bandalart.android.core.ui.component
+
+import android.widget.Toast
+import androidx.compose.foundation.layout.height
+import androidx.compose.material3.FilledIconButton
+import androidx.compose.material3.IconButtonColors
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.unit.dp
+import com.nexters.bandalart.android.core.ui.theme.Gray200
+import com.nexters.bandalart.android.core.ui.theme.Gray400
+import com.nexters.bandalart.android.core.ui.theme.Gray900
+import com.nexters.bandalart.android.core.ui.theme.White
+
+@Composable
+fun BottomSheetDeleteButton(
+  modifier: Modifier = Modifier,
+) {
+  val contextForToast = LocalContext.current.applicationContext
+  FilledIconButton(
+    onClick = { Toast.makeText(contextForToast, "삭제 버튼", Toast.LENGTH_SHORT).show() },
+    colors = IconButtonColors(Gray200, Gray900, Gray200, Gray900),
+    modifier = modifier
+      .height(55.dp),
```

Comment:
맞다 내가 말을 안했는데 modifier 속성이 한개 뿐이면 그냥 한줄에다 하는게 좋을거 같애 두개 이상이면 이런식으로 개행하구 
```suggestion
modifier.height(55.dp),
```

Reply:
이건 제일 최근거에도 이렇게 되어있었네 수정했어!